src/main/scala/dameo/format/Forsyth.scala

```diff
@@ -3,169 +3,107 @@ package format
 
 import cats.implicits._
 
-import scala.util.Try
-
 import scala.annotation.nowarn
 
 import strategygames.Player
 
 import variant.{ Dameo, Variant }
 
-//TODO Dameo Need to rewrite a lot of this
-//Most of this has been copied out of Draughts gamelogic
-//To demonstrate what goes on here, see also FEN.scala
-
 /** Transform a game to draughts Forsyth Edwards Notation
-  * https://en.wikipedia.org/wiki/Portable_Draughts_Notation Additions: Piece role G/P = Ghost man or king of
-  * that player, has been captured but not removed because the forced capture sequence is not finished yet
-  * ":Hx" = Halfmove clock: This is the number of halfmoves since a forced draw material combination appears.
-  * This is used to determine if a draw can be claimed. ":Fx" = Fullmove number: The number of the full move.
-  * It starts at 1, and is incremented after P2's move.
+  * https://en.wikipedia.org/wiki/Portable_Draughts_Notation Additions: Piece role g/p = Ghost man or king of
+  * that player, has been captured but not removed because the forced capture sequence is not finished yet.
+  * Piece role a/b = Active Man or King, the piece performing the forced capture sequence in progress. ":Hx" =
+  * Halfmove clock: This is the number of halfmoves since a forced draw material combination appears. This is
+  * used to determine if a draw can be claimed. ":Fx" = Fullmove number: The number of the full move. It
+  * starts at 1, and is incremented after P2's move.
   */
 object Forsyth {
 
-  val initial              =
-    FEN(
-      "W:W31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50:B1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20:H0:F1"
-    )
-  val initialPieces        =
-    FEN(
-      "W31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50:B1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20"
-    )
-  val initialMoveAndPieces =
+  val initial =
     FEN(
-      "W:W31,32,33,34,35,36,37,38,39,40,41,42,43,44,45,46,47,48,49,50:B1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20"
+      "W:Wa1,b1,b2,c1,c2,c3,d1,d2,d3,e1,e2,e3,f1,f2,f3,g1,g2,h1:Ba8,b7,b8,c6,c7,c8,d6,d7,d8,e6,e7,e8,f6,f7,f8,g7,g8,h8:H0:F1"
     )
 
-  private def parseIntOption(str: String): Option[Int] =
-    Try(Integer.parseInt(str)).toOption
-
-  def <<@(variant: Variant, fen: FEN): Option[Situation] =
-    makeBoard(variant, fen) map { board =>
-      val situation = Player.apply(fen.value.charAt(0)) match {
-        case Some(player) => Situation(board, player)
-        case _            => Situation(board, P1)
-      }
-
-      situation withHistory {
+  def <<@(variant: Variant, fen: FEN): Option[Situation] = {
+    /* Convert a FEN + Variant into a Situation */
+    Some(
+      Situation(
+        Board(
+          pieces = fen.pieces,
+          history = History(halfMoveClock = fen.halfMoveClock.getOrElse(0)),
+          variant = variant
+        ),
+        fen.player.get
```

Comment:
dont want this .get. We can easily pull this out to make it `fen.player.map{p => Situation(Board(...), p).withHistory(...)}`